crates/bws/src/command/mod.rs

```diff
@@ -0,0 +1,61 @@
+pub(crate) mod project;
+pub(crate) mod secret;
+
+use std::{path::PathBuf, str::FromStr};
+
+use bitwarden::auth::AccessToken;
+use clap::CommandFactory;
+use clap_complete::Shell;
+use color_eyre::eyre::{bail, eyre, Result};
+
+use crate::{config, Cli, ProfileKey};
+
+pub(crate) fn completions(shell: Option<Shell>) -> Result<()> {
+    let Some(shell) = shell.or_else(Shell::from_env) else {
+        return Err(eyre!(
+            "Couldn't autodetect a valid shell. Run `bws completions --help` for more info."
+        ));
```

Comment:
Small nit: Instead of doing `return Err(eyre!())`, you can use the macro `bail!` which does the same.

```suggestion
        bail!("Couldn't autodetect a valid shell. Run `bws completions --help` for more info.");
```

Reply:
Literally in the [docs](https://docs.rs/eyre/0.6.12/eyre/macro.bail.html) 😂. Thanks, fixed in [146db57](https://github.com/bitwarden/sdk/pull/809/commits/146db574cef4b2bc734d0e876f1b7c14b02258dd).